apps/meteor/tests/e2e/imports.spec.ts

```diff
@@ -43,11 +76,45 @@ test.describe.serial('imports', () => {
 		});
 	});
 
-	test('expect to all users imported are actually listed as users', async ({ page }) => {
+	test('expect import users data from zipped CSV files', async ({ page }) => {
+		const poAdmin: Admin = new Admin(page);
+		await page.goto('/admin/import');
+
+		await poAdmin.btnImportNewFile.click();
+
+		await (await poAdmin.getOptionFileType('CSV')).click();
+
+		await poAdmin.inputFile.setInputFiles(zipCsvImportDir);
+		await poAdmin.btnImport.click();
+
+		await poAdmin.btnStartImport.click();
+
+		await expect(poAdmin.importStatusTableFirstRowCell).toBeVisible({
+			timeout: 30_000,
+		});
+	});
+
+	test('expect all imported users to be actually listed as users', async ({ page }) => {
 		await page.goto('/admin/users');
 
 		for (const user of rowUserName) {
 			expect(page.locator(`tbody tr td:first-child >> text="${user}"`));
 		}
 	});
+
+	test('expect all imported rooms to be actually listed as rooms', async ({ page }) => {
+		const poAdmin: Admin = new Admin(page);
+		await page.goto('/admin/rooms');
+
+		for await (const room of importedRooms) {
+			await poAdmin.inputSearchRooms.fill(room.name);
+
+			const expectedMembersCount = room.members.split(';').filter((username) => username !== room.ownerUsername).length + 1;
+			expect(page.locator(`tbody tr td:nth-child(2) >> text="${ expectedMembersCount }"`));
+
+			await poAdmin.getRoomRow(room.name).click();
+			room.visibility === 'private' ? await expect(poAdmin.privateInput).toBeChecked() : await expect(poAdmin.privateInput).not.toBeChecked();
+			await expect(poAdmin.roomOwnerInput).toHaveValue(room.ownerUsername);
```

Comment:
I believe it would be better to have this in another test case, it could be a separate test case to cover the room members, so we can check the members count as you did above + check the room owner.